### DISCUSSION TITLE
Ask any questions you have on documentation

### DISCUSSION THREAD

**Discussion by MykolaGolubyev** [testingisdocumenting/znai#846] [Q&A]
Let me know any questions you have on documentation in general and about Znai in particular.

**Comment by baobab-it** [comment]
Hello

After reading documentation, it's amazing work you do. Here are my wishes for the improvement of Znai.

1. Update PlantUML version

I've checked current version:

```plantuml
@startuml
version
@enduml
```

Using version is almost 2 years old. Can you update library to latest version?

2. Add title for left menu and attention signs

Sometime not all text can see in left side menu, can you add attribute title to show all text of topic?

The same in attention signs if I use two sides pages more natural only show "note", "question", ... in title.

Or can add some option for attention signs, something like that:

```
Note:{title} Some text
```

If the option set - use title, otherwise - show label.

3. Add attention sign "Recommendation"

Znai have the attention sign "Don't:" but there are no opposite attention signs, like: "Recommendation:", "Tips and tricks:", etc. The `check-circle` icon is very good for this attention sign.

4. How to create anchor for div, tables, etc.?

We can create link from header, but in my case, I have use intensive linking in page using paragraphs, images, table. Can you create some plugin for creation anchor?

```html
<div id="some-unique-id" ...
<img id="some-unique-id" ...
<table id="some-unique-id" ...
```

5. Optional, add database query language syntax highlighting: SQL, Cypher (in future GQL)

**Reply by MykolaGolubyev** [reply]
Thank you
This is a great list! Any chance you can create independent issues?
If possible, could you elaborate on title for left panel?

**Comment by baobab-it** [comment]
Sure, I'll create issues.

About left panel. An example of a problem, in `Znai User Guide` open page "Getting Started" and in bottom sub panel "On-site Central Hub Deploym..." (text cutting and display `...`). It will be nice if I'm hovering the mouse on the link display title with full text description, like in a search box with title "hotkey /".

**Reply by MykolaGolubyev** [reply]
got it, makes sense. I thought it is already doing it. Should be an easy fix
